04_thermal_conductivity/README.md

```diff
@@ -0,0 +1,456 @@
+# Tutorial for the Thermal Conductivity
+ 
+In this tutorial, you will learn how to compute lattice thermal conductivity in solids using the TDEP method. 
+Useful references on the topic:
+
+[R.E. Peierls, Quantum Theory of Solids (1955)](https://books.google.es/books?id=WvPcBUsSJBAC&redir_esc=y)
+
+
+[M. Omini and A. Sparavigna Phys. Rev. B 53, 9064 (1996)](https://journals.aps.org/prb/abstract/10.1103/PhysRevB.53.9064)
+
+
+[D. A. Broido et al., Appl. Phys. Lett. 91, 231922 (2007)](https://pubs.aip.org/aip/apl/article-abstract/91/23/231922/334217/Intrinsic-lattice-thermal-conductivity-of?redirectedFrom=fulltext)
+
+# General scope
+In this tutorial we will learn how to calculate the lattice thermal conductivity from the iterative solution of the phonon Boltzmann equation.
+
+The tutorial covers:
+* the basic features included in the thermal conductivity routine of TDEP
+* thermal conductivity as a function of temperature
+* thermal conductivity for a given isotope distribution
+* thermal conductivity vs q-point grid.
+* plot and analysis of the results
+
+The tutorial **does not cover**:
+* Relax the structure
+* Supercell convergence
+* Extract force constants
+* Convergence of the results
+
+  
+There are two test cases: Al and MgO. 
+The data provided includes the IFCs and the unitcell obtained from previous DFT calculations. This data is not meant to produce converged results. 
+
+# Before running the tutorial:
+* TDEP installed
+* [H5PY](https://docs.h5py.org/en/stable/) for Python installed
+* Access to a plotting tool (for example  [matplotlib](https://matplotlib.org/), if you are using Python)
+* Have a converged set of 2nd and 3rd order IFCs (you can use some previous examples or the data provided in this tutorial)
+
+## Input files:
+  `infile.ucposcar` 
+  `infile.forceconstants` 
+  `infile.forceconstant_thirdorder` 
+  
+## Optional input file:
+  `infile.isotopes` (for non-natural isotope distribution)
+  
+You can create your customized isotope distribution specifying the number of isotopes per atom, followed by the appropriate number of concentrations and masses (in atomic mass units). An example is reported below:
+
+```
+1         # number of isotopes for first atom in infile.ucposcar
+1 28.0855 # concentration, mass, one line per isotope
+2         # number of isotopes for second atom
+0.5 12.0  # concentration, mass
+0.5 13.0  # concentration, mass
+...
+```
+
+# Basic steps
+
+## Preparation
+
+Read the documentation for the [thermal conductivity](https://ollehellman.github.io/program/thermal_conductivity.html)
```

Comment:
please update link to new webpage: https://tdep-developers.github.io/tdep/program/thermal_conductivity/